frontend/src/routes/(authenticated)/project/create/+page.svelte

```diff
@@ -100,6 +110,13 @@
     }, { taint: false });
   });
 
+  // as soon as code becomes invalid, explicit validation becomes redundant
+  const codeValidationDebouncer = debounce(() => {
+    void validate('code');
+    validateCode = () => {};
+  }, 500);
+  let validateCode = codeValidationDebouncer.debounce;
```

Comment:
huh? I have no idea what's going on here. After a debounce it'll never call validate explicitly again. Doesn't matter if it's invalid or not.